release-notes/opensearch-anomaly-detection.release-notes-2.9.0.0.md

```diff
@@ -0,0 +1,23 @@
+Compatible with OpenSearch 2.9.0.
+
+### Enhancements
+
+* Refactor Shared Settings, Introduce Dynamic Forecast Settings, Rename ([#912](https://github.com/opensearch-project/anomaly-detection/pull/912))
+* Refactoring Index Creation for Improved Code Reuse and Consistency ([#932](https://github.com/opensearch-project/anomaly-detection/pull/932))
+* Add ForecastResult and Refactor Shared Code ([#941](https://github.com/opensearch-project/anomaly-detection/pull/941))
+* Refactor: Migrate files from 'ad' to 'timeseries' package ([#918](https://github.com/opensearch-project/anomaly-detection/pull/918))
+* Add Forecaster class ([#920](https://github.com/opensearch-project/anomaly-detection/pull/920))
+* Refactor: Reorganize code for Forecasting and AnomalyDetector packages ([#925](https://github.com/opensearch-project/anomaly-detection/pull/925))
+* Enforce DOCUMENT Replication for AD Indices and Adjust Primary Shards ([#948](https://github.com/opensearch-project/anomaly-detection/pull/948))
```

Comment:
There are not in 2.9 release. You can remove them.

Reply:
I meant the following changes are not in 2.9:

* Refactor Shared Settings, Introduce Dynamic Forecast Settings, Rename ([#912](https://github.com/opensearch-project/anomaly-detection/pull/912))
* Refactoring Index Creation for Improved Code Reuse and Consistency ([#932](https://github.com/opensearch-project/anomaly-detection/pull/932))
* Add ForecastResult and Refactor Shared Code ([#941](https://github.com/opensearch-project/anomaly-detection/pull/941))
* Refactor: Migrate files from 'ad' to 'timeseries' package ([#918](https://github.com/opensearch-project/anomaly-detection/pull/918))
* Add Forecaster class ([#920](https://github.com/opensearch-project/anomaly-detection/pull/920))
* Refactor: Reorganize code for Forecasting and AnomalyDetector packages ([#925](https://github.com/opensearch-project/anomaly-detection/pull/925))

Reply:
Thanks - I've removed